code/modules/vehicles/bike.dm

```diff
@@ -51,12 +51,12 @@
 	if(usr.incapacitated()) return
 
 	if(kickstand)
-		src.visible_message("You put up \the [src]'s kickstand.")
+		src.visible_message("\The [usr] puts up \the [src]'s kickstand.")
 	else
 		if(istype(src.loc,/turf/space))
 			usr << "<span class='warning'> You don't think kickstands work in space...</span>"
 			return
-		src.visible_message("You put down \the [src]'s kickstand.")
+		src.visible_message("\The [usr] puts down \the [src]'s kickstand.")
```

Comment:
What about using this?
 `src.visible_message("\The [usr] puts up \the [src]'s kickstand.", "You put up \the [src]'s kickstand.", "You hear a thunk.")`